jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java

```diff
@@ -73,7 +73,7 @@ public void run() {
             flush();
           }
         },
-        0,
+        flushInterval,
```

Comment:
The version of `schedule()` with just `delay` argument runs just once (after specified delay) - both are needed here.